geomstats/geometry/spd_matrices.py

```diff
@@ -623,6 +623,35 @@ def geodesic(self, initial_point, initial_tangent_vec):
             initial_tangent_vec=initial_tangent_vec,
             point_type='matrix')
 
+    def parallel_transport(self, tangent_vec_a, tangent_vec_b, base_point):
+        r"""Parallel transport of a tangent vector.
+
+        Closed-form solution for the parallel transport of a tangent vector a
+        along the geodesic defined by exp_(base_point)(tangent_vec_b).
+        Denoting `tangent_vec_a` by `S`, `base_point` by `A`, let
+        `B = Exp_A(tangent_vec_b)` and :math: `E = (BA^{- 1})^({ 1 / 2})`.
+        Then the
+        parallel transport to `B`is:
+
+        ..math::
+                        S' = ESE^T
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[n_samples, dimension + 1]
```

Comment:
Missing parameters' description. I have filled one on parallel_transport in PR #425 , maybe copy paste this one?